polaris-react/src/components/Badge/Badge.scss

```diff
@@ -15,7 +15,7 @@
   font-weight: var(--p-font-weight-regular);
 
   @media print {
-    border: solid 0.1px var(--p-border);
```

Comment:
Tested changing this value from `0.1px` to `1px` (via out tokens) and didn't see a visual change when printing the page:
<img width="109" alt="Screen Shot 2022-04-12 at 2 57 55 PM" src="https://user-images.githubusercontent.com/21976492/163492633-1c3202a4-f7f1-4b70-abcf-2f65b533b542.png">
<img width="107" alt="Screen Shot 2022-04-12 at 2 58 29 PM" src="https://user-images.githubusercontent.com/21976492/163492639-63dc41ff-72a0-4848-818e-5f69a63d3c18.png">